lib/core/src/server/config.js

```diff
@@ -7,6 +7,7 @@ function wrapCorePresets(presets) {
   return {
     babel: (config, args) => presets.apply('babel', config, args),
     webpack: (config, args) => presets.apply('webpack', config, args),
+    webpackFinal: (config, args) => presets.apply('webpackFinal', config, args),
```

Comment:
Since I am not familiar with the new presets: Can developers access the presets from outside? 
I was just thinking that `webpack` seems irrelevant because `webpackFinal` is the real working webpack configuration

Do we even need 2 presets or just apply `webpackFinal` on `webpack` in the background and only offer 1 webpack preset? Or is this some sort of "if anything went wrong, `webpack` is your safety net"?

Reply:
> Can developers access the presets from outside 

It's possible with the [custom presets](https://github.com/storybooks/storybook/blob/master/lib/core/src/server/config.js#L15) (currently experimental, but will be not experimental after the #4169)

Presets are chained. Let's say you have:

``` 
preset-a
  webpack
  webpackFinal
 
preset-b
  webpack

preset-c
  webpack
  webpackFinal
```
they will be called in the next order: 

1. preset-a.webpack -> preset-b.webpack -> preset-c.webpack
2. preset-a.webpackFinal -> preset-c.webpackFinal

That way some earlier defined preset can override the configuration of the later defined preset, of course if the later one doesn't have its own extension.

I was thinking about having something like `pitch` api in webpack loaders (they have it in the opposite order).

So it will be like this:

```
preset-a.webpack 
       preset-c.webpack
              preset-d.webpack
              preset-d.webpackBack
       preset-c.webpackBack     
preset-a.webpackBack
```
It can be supported in the [core level](https://github.com/storybooks/storybook/blob/master/lib/core/src/server/presets.js#L44)